content/en/docs/installation/compatibility.md

```diff
@@ -5,9 +5,18 @@ weight: 100
 type: "docs"
 ---
 
+
 Below you will find details on various compatibility issues and quirks that you
 may be effected by in your environment.
 
+- [GKE](#gke)
+- [AWS EKS](#aws-eks)
+- [Webhook](#webhook)
+- [iptables vs. nftables](#iptables-vs-nftables)
+  - [Switch to `iptables`](#switch-to-iptables)
+  - [Configure network provider](#configure-network-provider)
+    - [Calico](#calico)
```

Comment:
Please remove this extra ToC.
There is already a [document ToC on the right](https://deploy-preview-187--cert-manager.netlify.app/docs/installation/compatibility/) which is maintained automatically.

Reply:
Removed the toc, please have a look again